templates/navbar.html

```diff
@@ -3,7 +3,7 @@
         <div class="container mx-auto ">
             <div class="flex flex-wrap justify-between items-center">
                 <div class="flex justify-start items-center">
-                    <a href="#" class="flex mr-4">
+                    <a href="{% if config.extra.logo_redirect_to %}{{config.extra.logo_redirect_to}}{% else %}/{% endif %}" class="flex mr-4">
```

Comment:
Is it possible to make the fallback the `base_url`, instead of the domain root? That seems less likely to require an explicit redirect in the config...

Reply:
Done